src/io.F

```diff
@@ -947,6 +947,117 @@ subroutine printchecks(idump)
       return
       end
 c-----------------------------------------------------------------------
+      subroutine errors_output(out1,out2,out3,out4,out5,out6,
```

Comment:
Also I wonder if there is really an advantage to a routine that computes the errors and then prints them versus a routine which computes the errors (which already exists in `utilities.F`) plus a `write` statement. If you split it into two steps then there's no need to guess at how many arrays you need to print errors for, which will invariably change for different applications.